cli/config/configuration-qemu-virtiofs.toml.in

```diff
@@ -466,3 +466,17 @@ sandbox_cgroup_only=@DEFSANDBOXCGROUPONLY@
 # Supported experimental features:
 # (default: [])
 experimental=@DEFAULTEXPFEATURES@
+
+# Use VFIO driver for VFIO devices in the VM, as well as the host.
+#
+# When enabled, VFIO devices passed to the container appear as VFIO
+# devices within the container, similar behaviour to a conventional
+# container.  NOTE: The VFIO device within the container will have a
+# different name from the device on the host, since the VM has its own
+# IOMMU/VFIO group numbering.
+#
+# When disabled, the VM will use its native drivers for the VFIO
+# device.This behaviour is inherently Kata specific, and will require
+# a container image which expects it.
+# (default: false)
+#vfio_in_guest = true
```

Comment:
Having received that feedback from @fidencio not very long ago, I would recommend you add a variable to the `Makefile` and add something that looks like `@DEFVIRTIOFSINGUEST@` in the `.in` files. That way, the various distros can more easily define their own default values.

Something like:
```
# default if not set: false
# Your distribution recommends: @DEFVIRTIOFSINGUEST@
vfio_in_guest = @DEFVIRTIOFSINGUEST@
```